### ISSUE TITLE
Allow for custom lists

### ISSUE THREAD

**Issue by EricHigdon** [EriQ/ConceptForge#2]
Add programming to allow for a user-generated list. We need to decide what the best method of input would be. Should we have them put a comma, separated list into one big text box? Or each item goes into its own text box?

Perhaps there is a way we could do both and have the user decide how they want to input their lists.

**Comment by EriQ** [comment]
Lists should be entered in a text are separated by a return character (each item on a new line).

**Comment by TimKellyTheMighty** [comment]
If this is going to stay with the update from website to app, you may want make it a phase 2 project.

**Comment by EriQ** [comment]
Agreed. I moved it to phase 2 for now. I would still like to get this implemented soon, but I'm not sure how practical it will be to type a list on mobile devices.
